### PR TITLE
1690 (advanced form improvements)

### PR DESCRIPTION
**Pull request by lilyhoutman**
Description:
1690 (advanced form improvements) combines many minor JIRA tickets all related to the advanced form

- 1661 (Add new 'Results Filter' section to forms)
- 1663 (Develop tips for download types and sort)
- 1665 (Redefine "Copy to Clipboard")
- 1675 (Somehow tell users that no fields are required for any search)
- 1687 (Adjust tooltip width)
- 1688 (Adjust CSS for select2 inputs to look like USWDS inputs)

Before making a pull request
----------------------------

- [x] Make sure all tests run
- [x] Update the changelog appropriately

Title
-----------
Brief description of changes. Reference the JIRA ticket if appropriate

Description
-----------
If no JIRA ticket is referenced, describe the changes made. Note anything that you want the reviewers to know while
reviewing your pull request

After making a pull request
---------------------------
- [x] If appropriate, put the link to the PR in the JIRA ticket
- [x] Assign someone to review unless the change is trivial

### CODE REVIEW

**Comment by chopkins-usgs** [comment]
No changes suggested.

File: assets/scss/custom/custom.scss [hunk 1]
```diff
@@ -201,3 +195,51 @@ $black: #000;
 #a3 {
   overflow: visible
 }
+#a4 {
```
Comment (mbucknell) [inline assets/scss/custom/custom.scss:198]: I am not a big fan of generic id names. I think this is to style an accordian. I would suggest creating a class that can be applied to accordians that we want to apply this style to and then change the markup/style sheet to use the class.

File: assets/scss/custom/custom.scss [hunk 2]
```diff
@@ -201,3 +195,51 @@ $black: #000;
...
+  min-height: 26px !important;
+}
+
+.select2-selection__choice {
+  height: 20px !important;
+  margin: 0px !important;
+}
+
+.usa-textarea {
+  height: 5rem !important;
```
Comment (mbucknell) [inline assets/scss/custom/custom.scss:231]: This is another area where I think creating a wqp-textarea class and then applying the style to that and using the class when we want a text-area to have this height would be good.

File: server/wqp/portal_ui_blueprint/templates/portal.html [hunk 3]
```diff
@@ -20,59 +19,59 @@ <h4 class="usa-accordion__heading">
...
+                <div>Use the location parameters to describe the spatial extent of the desired dataset.
+                </div>
+                <br>
                 <div class="grid-row grid-gap">
-                  <div class="tablet:grid-col">
+                  <div class="tablet:grid-col-3">
                     <div class="form-group">
-                      <label for="countrycode" class="tooltip" :title="countryTooltip">Select one or multiple
-                        countries:</label>
+                      <label><strong>Country </strong></label><label class="tooltip" class="control-label" for="control-label" :title="countryTooltip"><span style="color: #565c65
```
Comment (mbucknell) [inline server/wqp/portal_ui_blueprint/templates/portal.html:28]: At some point, I think we need to switch to using USWDS classes to style the form labels and elements. May not be appropriate for the PR but we should at some point.

File: server/wqp/portal_ui_blueprint/templates/portal.html [hunk 4]
```diff
@@ -513,103 +549,84 @@ <h2 class="usa-modal__heading" id="modal-2-heading">
...
+  <div class="modal-footer">
+    <button id="validate-close-button" type="button" class="btn btn-default" data-dismiss="modal">
+      Ok
+    </button>
+  </div>
+</div>
+</div>
+</div> -->
+{% endblock %} {% block body_page_script %}
+<script src="https://cdn.jsdelivr.net/npm/vue@2/dist/vue.js"></script>
```
Comment (mbucknell) [inline server/wqp/portal_ui_blueprint/templates/portal.html:630]: I'd like to eventually but vue in the bundle. Now that I have some experience with this, I can more intelligently help with this. This doesn't need to be done now.

File: server/wqp/portal_ui_blueprint/templates/portal.html [hunk 5]
```diff
@@ -513,103 +549,84 @@ <h2 class="usa-modal__heading" id="modal-2-heading">
...
+  </div>
+  <div class="modal-footer">
+    <button id="validate-close-button" type="button" class="btn btn-default" data-dismiss="modal">
+      Ok
+    </button>
+  </div>
+</div>
+</div>
+</div> -->
+{% endblock %} {% block body_page_script %}
```
Comment (mbucknell) [inline server/wqp/portal_ui_blueprint/templates/portal.html:629]: we should clean up this formatting. In Jinja land blocks should start on their own line with an empty line between the block and the preceding one.

**Review by mbucknell — APPROVED** [review]
Made several suggestions but nothing that would hold this up.

File: assets/scss/custom/custom.scss (hunk 1 above)
Comment (lilyhoutman) [reply assets/scss/custom/custom.scss]: renamed!

File: assets/scss/custom/custom.scss (hunk 2 above)
Comment (lilyhoutman) [reply assets/scss/custom/custom.scss]: class added!

File: server/wqp/portal_ui_blueprint/templates/portal.html (hunk 3 above)
Comment (lilyhoutman) [reply server/wqp/portal_ui_blueprint/templates/portal.html:28]: working on in a new branch!

File: server/wqp/portal_ui_blueprint/templates/portal.html (hunk 5 above)
Comment (lilyhoutman) [reply server/wqp/portal_ui_blueprint/templates/portal.html]: updated this specific one - I didn't touch anything with blocks, so others may still be incorrectly formatted

**Review by lprivette — APPROVED** [review]

**Pull request merged**